classes/class-twentytwenty-color.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * Color calculations.
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty
+ * @since 1.0.0
+ */
+
+if ( ! class_exists( 'TwentyTwenty_Color' ) ) {
+	/**
+	 * Color Calculations.
+	 */
+	class TwentyTwenty_Color {
+
+		/**
+		 * The HEX color.
+		 *
+		 * @access private
+		 * @since 1.0
```

Comment:
When since strings are specified could we use 3 digit references always? - `1.0.0`